css/my_portfolio.css

```diff
@@ -352,3 +351,83 @@ button:active {
 .margin-05 {
   margin: 0 5%;
 }
+
+#contact-us {
+  height: 100vh;
+  padding-top: 30%;
+  text-align: center;
+}
+
+.contact-us-text {
+  width: 100%;
+  text-align: center;
+  padding: 0 7%;
+}
+
+.contact-us-paragraph {
+  text-align: center;
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 27px;
+  line-height: 40px;
+}
+
+.form {
+  text-align: center;
+  padding: 8%;
+}
```

Comment:
- [ ] Kindly use `display: flex` for this class property because it is a major requirement to style the form area with _**flexbox**_.

Reply:
Thanks 😘...I think `display: flex` is not for `form` it is for media- icons below it like github, twitter, LinkedIn.. 😑